flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobResult.java

```diff
@@ -133,26 +132,32 @@ public JobExecutionResult toJobExecutionResult(ClassLoader classLoader) throws J
 					accumulatorResults,
 					classLoader));
 		} else {
-			final Throwable cause;
```

Comment:
This `else()` block can now become simply:
`throw getJobExecutionException(serializedThrowable, classLoader);` right? We do not need to wrap the original exception anymore.

Reply:
@kl0u Yes, your point is right for me. I forgot to remove the unnecessary wrapper for `JobExecutionException`. I would like to remove this wrapper.